tests/baselines/reference/thisTypeInFunctions.js

```diff
@@ -243,7 +243,8 @@ function implicitThis(n) {
 var impl = {
     a: 12,
     explicitVoid2: function () { return _this.a; },
-    explicitVoid1: function () { return 12; },
+    explicitVoid1: // ok, this: any because it refers to some outer object (window?)

```

Comment:
this seems like regression ? Should the comment be there ? In the (original)[https://github.com/Microsoft/TypeScript/pull/8197/files#diff-244ce633ddb2ae835e68b8278cbf7e32R41] it should be after `explicitVoid2`